app/javascript/packs/application.js

```diff
@@ -17,6 +19,11 @@ import '../blacklight_overrides';
 
 require.context('../psulib_blacklight/images/', true);
 document.addEventListener('DOMContentLoaded', () => {
+  Rails.start();
+  Turbolinks.start();
+});
+
+document.addEventListener('turbolinks:load', () => {
```

Comment:
I think we removed these so the 'Ask a Librarian' widget would show properly.  Just a heads up in case that gets affected by this change.

Reply:
right, it was for Ask a Librarian in bookmarks page. Availability started to fail and I could get it work by adding them back, it can be due to another underlying thing. I am kinda thinking ask a librarian on bookmarks are a bit less of an issue so maybe we can take a look at this again in the next sprint.